frontend/pages/docs/users/programmatically-using-bailo/authentication.mdx

```diff
@@ -24,15 +24,15 @@ Client("http://127.0.0.1:8080", cert="path/to/cert.pem", key="path/to/key.pem",
 
 ## Token Agents
 
-If not using PKI Bailo allows the usage of access tokens to allow for fine-grained access Tokens are used to grant
-access to models within bailo to specific models and specific actions. These can be found in
+If you are not using PKI then Bailo allows the usage of access tokens to allow for fine-grained access. Tokens are used
```

Comment:
This is not actually accurate, Bailo allows tokens all the time!

Reply:
I just edited this as it was an existing doc, how would you phrase it instead?